lib/routes/drivers/routes.js

```diff
@@ -0,0 +1,130 @@
+            try {
+                let driver = await addDriver(request.payload);
+                ['createdAt', 'updatedAt'].forEach(key => delete driver[key]);
+                return h.response(driver).code(200);
+            } catch (error) {
+                return h
+                    .response({ status: 400, message: error.message })
+                    .code(400);
+            }
+        }
+    },
+    {
+        method: 'PUT',
+        path: '/profile/{driverId}',
+        options: {
+            description: 'update driver profile.',
+            notes: 'PUT drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: drivers.profile
+        },
+        handler: async (request, h) => {
+            try {
+                const id = request.params.driverId;
+                request.payload.id = id;
+                await updateDriver(request.payload);
+                return h
+                    .response('Driver profile updated successfully.')
+                    .code(200);
+            } catch (error) {
+                return h
+                    .response({ status: 400, message: error.message })
+                    .code(400);
+            }
+        }
+    },
+    {
+        method: 'POST',
+        path: '/add/location',
+        options: {
+            description: 'add driver location.',
+            notes: 'POST drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: drivers.location
+        },
+        handler: async (request, h) => {
+            try {
+                const { currentLocation } = request.payload;
+                request.payload.currentLocation = Sequelize.fn(
+                    'ST_GeomFromText',
+                    `POINT(${currentLocation[0]} ${currentLocation[1]})`
+                );
+                let driverLocation = await addDriverLocation(request.payload);
+                return h.response(driverLocation).code(200);
+            } catch (error) {
+                return h
+                    .response({ status: 400, message: error.message })
+                    .code(400);
+            }
+        }
+    },
+    {
+        method: 'POST',
+        path: '/accept',
+        options: {
+            description: ' accept the booking.',
+            notes: 'POST drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: drivers.accept
+        },
+        handler: async (request, h) => {
+            try {
+                await acceptBooking(request.payload);
+                let bookingAttemptPayload = {
+                    bookingAttemptId: request.payload.bookingAttemptId,
+                    rideStatus: 'accepted'
+                };
+                await updateBookingAttempts(bookingAttemptPayload);
+                let bookingAttempt = convertDbResponseToRawResponse(
```

Comment:
Not a big fan of doing this conversions here I believe in create an helper in dao and using it there.

Reply:
Made the changes as suggested.